src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -91,7 +91,7 @@ const UsersFragment = ({
   const [client] = React.useContext(Context).client;
   const {exhangeFollower, users, updateFollowDiscoveryContext} = useDiscovery();
   const [loadingDM, setLoadingDM] = React.useState(false);
-  const {createSignChat} = useCreateChat();
+  const {createSignChat, loadingCreateChat} = useCreateChat();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `loadingCreateChat` from `useCreateChat()` is a good move as it allows the UI to respond to the loading state of chat creation. This can improve user experience by providing feedback when the chat is being created.
```diff
-  const {createSignChat} = useCreateChat();
+  const {createSignChat, loadingCreateChat} = useCreateChat();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->